doc/swagger-api-description/api.yaml

```diff
@@ -173,69 +176,214 @@ paths:
           schema:
             $ref: '#/definitions/ErrorModel'
 
-definitions:
+  /users/{userUuid}/clouds:
```

Comment:
We wont have simple API. Please move path "/users/{userUuid}/clouds" to "/clouds" without any other change.